src/Avalonia.Controls/MaskedTextBox.cs

```diff
@@ -184,6 +184,32 @@ public bool ResetOnSpace
 
         Type IStyleable.StyleKey => typeof(TextBox);
 
+        public new string Text
```

Comment:
I think it's better like this:
```csharp
        protected override void OnPropertyChanged<T>(AvaloniaPropertyChangedEventArgs<T> change)
        {
            if (change.Property == TextProperty)
            {
                // Your code
            }
            base.OnPropertyChanged(change);
        }
```

Reply:
Yeah, this is going to cause surprising behavior if someone accesses the `Text` property via a reference to `TextBox` and not `MaskedTextBox`.

Reply:
Will also not be called when text is set by calling `maskedTextBox.SetValue(MaskedTextBox.TextProperty, value)`